certbot-ci/certbot_integration_tests/certbot_tests/test_main.py

```diff
@@ -567,6 +568,9 @@ def test_ocsp_status_live(context):
 
     # OSCP 2: Check live certificate OCSP status (REVOKED)
     context.certbot(['revoke', '--cert-name', cert, '--no-delete-after-revoke'])
+    # Sometimes in oldest tests (using openssl binary and not cryptography), the OCSP status is
+    # not seen immediately by Certbot as invalid. Waiting few seconds solves this transient issue.
+    time.sleep(5)
```

Comment:
This change came in while I was reviewing the PR and I missed it.

Instead of doing this, I'd rather fix the problem in Certbot? Are we not specifying enough wiggle room with validity times to OpenSSL?

Reply:
The problem is transient, only observed on Travis so far, not locally. I not know how to recreate a reliable pattern and find precisely what needs to be done.

I would like to avoid to block this PR on that, since it is unrelated. We could open an issue to track finding a way to troubleshoot and fix.

Reply:
Do you have a log of this failure?

Reply:
https://travis-ci.com/certbot/certbot/builds/111407674